test/cursor.test.ts

```diff
@@ -56,13 +57,13 @@ suite("cursor", () => {
 	});
 
 	test("right on right-most column should stay at the same location", () => {
-		Cursor.move(new vscode.Position(0, 7));
+		Caret.move(new vscode.Position(0, 7));
 
-		let current = Cursor.currentPosition();
+		let current = Caret.currentPosition();
 		assert.equal(current.line, 0);
 		assert.equal(current.character, 7);
 
-		let right = Cursor.right();
+		let right = Caret.right();
 		assert.equal(right.line, 0);
```

Comment:
Can we keep this test suite focused on the `Cursor` class? Keep using the Cursor functions but change the expected values on the asserts?